backends/bmv2/common/expression.cpp

```diff
@@ -204,7 +205,7 @@ void ExpressionConverter::postorder(const IR::ArrayIndex *expression) {
 
     if (!expression->right->is<IR::Constant>()) {
         const IR::Expression *ex = expression->right;
-        auto fresult = ::get(map, ex);
+        auto fresult = ::P4::get(map, ex);
```

Comment:
For the future: this likely should be just `get`, so we (potentially) might take advantage from ADL

Reply:
Yes -- as the comment in the header file says, this is mainly just providing a method missing from `std::map` that really should be there.